src/checkqueue.h

```diff
@@ -208,43 +208,35 @@ template <typename T, typename R = std::remove_cvref_t<decltype(std::declval<T>(
 class CCheckQueueControl
 {
 private:
-    CCheckQueue<T, R> * const pqueue;
+    CCheckQueue<T, R>& pqueue;
     bool fDone;
 
 public:
     CCheckQueueControl() = delete;
     CCheckQueueControl(const CCheckQueueControl&) = delete;
     CCheckQueueControl& operator=(const CCheckQueueControl&) = delete;
-    explicit CCheckQueueControl(CCheckQueue<T> * const pqueueIn) : pqueue(pqueueIn), fDone(false)
+    explicit CCheckQueueControl(CCheckQueue<T>& pqueueIn) : pqueue(pqueueIn), fDone(false)
```

Comment:
Nit: `s/pqueue/queue/`

The relevant lines are touched already, so maybe just do it in the same commit?

Reply:
double-nit: maybe `m_queue`

Reply:
Done.